lib/balance-tx/lib/internal/Internal/Cardano/Write/Tx.hs

```diff
@@ -463,13 +471,21 @@ instance Bounded AnyRecentEra where
     minBound = AnyRecentEra RecentEraBabbage
     maxBound = AnyRecentEra RecentEraConway
 
+instance Ord AnyRecentEra where
+    compare = compare `on` fromEnum
+
 instance Show AnyRecentEra where
     show (AnyRecentEra era) = "AnyRecentEra " <> show era
 
 instance Eq AnyRecentEra where
     AnyRecentEra e1 == AnyRecentEra e2 =
         isJust $ testEquality e1 e2
 
+-- | The complete set of supported recent eras.
+--
+supportedRecentEras :: Set AnyRecentEra
```

Comment:
> **supported**RecentEras

What would an unsupported recent era be?

It seems like it's saying the same things twice. I.e. could it be more suitable to name it `allRecentEras` or  `supportedErasForTxConstruction` (the latter more for the api, not here) ?

Reply:
@Anviking good point, thanks for this feedback!

I've adjusted this in 18a34f0d5cf05488499032f368a24cf5c5ed2e5b.